api/ce/kubelb.k8c.io/v1alpha1/tenant_state.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2025 The KubeLB Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// TenantStateSpec defines the desired state of TenantState.
+type TenantStateSpec struct {
+	// TenantState is used to represent the status of a tenant so it's spec is empty.
+}
+
+// TenantStateStatus defines the observed state of TenantState
+type TenantStateStatus struct {
+	Version     Version            `json:"version,omitempty"`
+	LastUpdated metav1.Time        `json:"lastUpdated,omitempty"`
+	Conditions  []metav1.Condition `json:"conditions,omitempty"`
+}
```

Comment:
Note for reviewer:

More fields will be added here later on for now we need the Version field to make a few features work